misk-web/web/packages/@misk/core/src/ducks/index.ts

```diff
@@ -1,8 +1,28 @@
+import { fromJS, List } from "immutable"
 export * from "./simpleDucksUtilities"
 export * from "./routerDucks"
 export * from "./simpleFormDucks"
 export * from "./simpleNetworkDucks"
 
+// from @misk/common/src/index.ts
+export interface IDefaultState {
+  data: any
+  error: any
+  loading: boolean
+  simpleTag: string
+  success: boolean
+  toJS?: () => any
+}
+
+export const createDefaultState = (simpleTag: string): IDefaultState =>
```

Comment:
can we add a comment documenting this if it's part of our API?

Reply:
Yes will do